modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -10,9 +10,9 @@
 
     <xsd:import namespace="http://www.w3.org/XML/1998/namespace"/>
     <xsd:import namespace="http://www.springframework.org/schema/beans"
-                schemaLocation="http://www.springframework.org/schema/beans/spring-beans-3.1.xsd"/>
+                schemaLocation="http://www.springframework.org/schema/beans/spring-beans-4.1.xsd"/>
     <xsd:import namespace="http://www.springframework.org/schema/context"
-                schemaLocation="http://www.springframework.org/schema/context/spring-context-3.1.xsd"/>
+                schemaLocation="http://www.springframework.org/schema/context/spring-context-4.1.xsd"/>
```

Comment:
yes, but that should be a different story to be done once we finish removing components on the branch